opt_out_http_api/run.py

```diff
@@ -1,5 +1,6 @@
 from api_methods import API
+from store.memory import OptOutMemory
 
-app = API()
+app = API(OptOutMemory)
```

Comment:
I'm guessing this doesn't work currently? It probably needs to be `app = API(OptOutMemory())`?